python-template/{{cookiecutter.placeholder_repo_name}}/setup.py

```diff
@@ -82,6 +100,13 @@ def is_requirement(line):
     keywords='Python edx',
     classifiers=[
         'Development Status :: 3 - Alpha',
+        {%- if cookiecutter.supports_django == "yes" %}
```

Comment:
Is it more accurate to say that in requires django than to say it supports django?

Reply:
yep, you are correct